voltron_list.yaml

```diff
@@ -0,0 +1,70 @@
+genericVoltron: &voltron
+    title: This is a compiled collection of god/recommended rolls from top community minds.
+    description: Thanks to u/Mercules904, u/pandapaxxy, u/HavocsCall, and @chrisfried for their contributions. Ping u/48klocs / @48klocs with any issues.
+definitions:
+    pandaPvP:
+        - panda_black_armory_pvp_picks.txt  
+        - panda_missing_black_armory_pvp.txt  
+        - panda_salvation_pvp.txt       
+        - panda_shadowkeep_pvp.txt
+        - panda_forsaken_pvp_rolls.txt     
+        - panda_penumbra_pvp.txt 
+        - panda_season_drifter_pvp.txt
+    pandaPvE:
+        - panda_black_armory_pve_picks.txt
+        - panda_missing_black_armory_pve.txt
+        - panda_forsaken_pve_rolls.txt
+        - panda_salvation_pve.txt
+        - panda_season_drifter_pve.txt
+        - panda_shadowkeep_pve.txt
+    pandaGeneric:
+        - panda_black_armory_picks.txt
+        - panda_dawn_ritual.txt
+        - panda_penumbra_pve.txt
+        - panda_pit_and_altar.txt
+    pandaMKB:
+        - panda_returning_dawn_mkb.txt
+        - panda_dawning_mkb.txt
+        - panda_sundial_mkb.txt
+    pandaController:
+        - panda_returning_dawn_controller.txt
+        - panda_dawning_controller.txt
+        - panda_sundial_controller.txt
+    mercules:
+        - mercules_altar_of_sorrow.txt
+        - mercules_black_armory_and_sotp_rolls.txt
+        - mercules_black_armory_pulse_bow.txt
+        - mercules_crown_of_sorrow.txt
+        - mercules_dreaming_city_breakdown_rolls.txt
+        - mercules_erentil_and_opulence_iron_banner.txt
+        - mercules_gambit_rolls.txt
+        - mercules_garden_of_salvation.txt
+        - mercules_iron_banner_bonus.txt
+        - mercules_iron_banner.txt
+        - mercules_last_wish_rolls.txt
+        - mercules_menagerie.txt
+        - mercules_osiris_redux.txt
+        - mercules_season_of_dawn_season_pass_breakdown.txt
+        - mercules_undying_season_pass_picks.txt
+        - mercules_vex_offensive_breakdown.txt
+    trash_lists:
+        - trash_rolls.txt
+        
+outputFiles:
+    voltron:
+        <<: *voltron
```

Comment:
Would you rather this be voltron_mkb that way voltron.txt can be your manually curated list? And later on you can decide whether or not you want to maintain classic voltron.txt vs auto-gen?

Reply:
Hmm, not sure yet. Ideally, I guess it should be scripted?

There looks like a lot of churn happened, so I'll have to review the file load order in the YAML to see if I can cut down on that.

Reply:
Put a comment later on in this about the churn. You can reduce the churn locally for doing a manual diff by doing `cat voltron_mkb.txt| sort -u  | grep -v '^//' > voltronMkbSorted` Which will sort all of the lines and then filter out comments (since comments in voltron and the source may differ and it doesn't matter). From there you can do a diff and then do a wc -l .

```
-[11:04:06][addAutoGenVoltron]$ cat voltron_mkb.txt| sort -u  | grep -v '^//' > voltronMkbSorted                                     ~/code/dim-wish-list-sources
-[11:04:10][addAutoGenVoltron?]$ cat voltron.txt| sort -u  | grep -v '^//' > voltronSorted                                           ~/code/dim-wish-list-sources
-[11:04:13][addAutoGenVoltron?]$ diff voltronSorted voltronMkbSorted|grep dimwishlist | wc -l                                        ~/code/dim-wish-list-sources
1041
```